massactions/tests/plugins/org.polarsys.kitalpha.massactions.junit.tests/tests/org/polarsys/kitalpha/massactions/tests/ju/test/view/ColumnHideShowTest.java

```diff
@@ -90,18 +91,16 @@ private void testColumnHideShow(MAView view) {
     ColumnHideShowLayer columnHideShowLayer = bodyLayer.getColumnHideShowLayer();
 
     // the internal representation of hidden columns
-    Set<Integer> beforeHiddenColumns = Collections.emptySet();
-    Set<Integer> afterHiddenColumns = Collections.emptySet();
+    Set<Integer> beforeHiddenColumns;
+    Set<Integer> afterHiddenColumns;
```

Comment:
To make the code easier to read and understund, can you please remove the declaration of the two sets from here and do them in the lines 118 and 124 (when the instances are created).